### PR TITLE
Update readme.md test

### PR DESCRIPTION
**Pull request by bselwe**

### CODE REVIEW

**Comment by bselwe** [comment]
@rafaelortizzableh test comment

**Comment by bselwe** [comment]
test

File: readme.md [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 ## Instructions
 
-To download Buzz, go into [Releases](https://github.com/joinbuzz/buzzer_download/releases), click on [`Download Buzzer - Latest`](https://github.com/joinbuzz/buzzer_download/releases/tag/download-latest-buzzer), and click on [`Buzzer.dmg`](https://github.com/joinbuzz/buzzer_download/releases/download/download-latest-buzzer/Buzzer.dmg) under `Assets`.
+To download Buzz, go into [Releases](https://github.com/joinbuzz/buzzer_download/releases), click on [`Download Buzzer - Latest`](https://github.com/joinbuzz/buzzer_download/releases/tag/download-latest-buzzer), and click on [`Buzzer.dmg`](https://github.com/joinbuzz/buzzer_download/releases/download/download-latest-buzzer/Buzzer.dmg) under `Assets`. Test.
```
Comment (bselwe) [inline readme.md:5]: Test

**Comment by bselwe** [comment]
test

**Comment by bselwe** [comment]
test

**Comment by bselwe** [comment]
test 2

File: readme.md (hunk 1 above)
Comment (bselwe) [reply readme.md:5]: test

**Comment by bselwe** [comment]
Test

**Comment by bselwe** [comment]
@rafaelortizzableh test mention

**Comment by bselwe** [comment]
Test

File: readme.md [hunk 2]
```diff
@@ -2,4 +2,4 @@
 
 ## Instructions
 
-To download Buzz, go into [Releases](https://github.com/joinbuzz/buzzer_download/releases), click on [`Download Buzzer - Latest`](https://github.com/joinbuzz/buzzer_download/releases/tag/download-latest-buzzer), and click on [`Buzzer.dmg`](https://github.com/joinbuzz/buzzer_download/releases/download/download-latest-buzzer/Buzzer.dmg) under `Assets`.
```
Comment (bselwe) [inline readme.md:5]: Test

**Comment by bselwe** [comment]
Test

**Comment by bselwe** [comment]
Test

File: readme.md (hunk 2 above)
Comment (ebbacc) [reply readme.md:5]: 123 test

Comment (ebbacc) [reply readme.md:5]: 123

**Review by ebbacc — CHANGES_REQUESTED** [review]

**Pull request closed without merging**